_posts/2016-09-10-stripe-dress.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: The psychology of perception tests an age-old fashion foible
+author: neuroamanda
+image:
+  feature: 2016-09-10-stripe-dress.jpg
+  credit: Gareth Williams
+  caption: Horizontal-striped dress
+---
+
+_Horizontal stripes on clothing are in fact slimming_
+
+[The dress](https://en.wikipedia.org/wiki/The_dress) sparked the hottest fashion debate of the 21st century, but another mystery has occupied scientists and fashionistas alike since the mid-19th century: do horizontal stripes make you look fat?
+The answer may seem obvious, but experimental results and a well-documented illusion contradict most people's intuition that horizontal stripes make the wearer look wider.
```

Comment:
Replace "most people's intuition" with "commonly held belief" -- I'm unsure if we have data on how many people have this intuition, so "most" is a bit misleading. I thought the research to gather data (here and elsewhere) did precisely the opposite.